test/roundrobin_test.go

```diff
@@ -88,6 +89,20 @@ func checkRoundRobin(ctx context.Context, cc *grpc.ClientConn, addrs []resolver.
 		}
 		iterations = append(iterations, iteration)
 	}
+	// Ensure the the first iteration contains all addresses in addrs. To
+	// support duplicate addresses, we determine the count of each address.
+	wantAddrCount := make(map[string]int)
+	for _, addr := range addrs {
+		wantAddrCount[addr.Addr]++
+	}
+	gotAddrCount := make(map[string]int)
+	for _, addr := range iterations[0] {
+		gotAddrCount[addr]++
+	}
+	if !cmp.Equal(gotAddrCount, wantAddrCount) {
```

Comment:
Use `Diff` for easier-to-see differences?

Reply:
Done.